src/main/java/ducky/Ducky.java

```diff
@@ -0,0 +1,54 @@
+package ducky;
+
+import ducky.command.Command;
+import ducky.util.Parser;
+
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * Represents a Ducky chatbot instance.
+ */
+public class Ducky {
+
+    private final Storage storage;
+    private final TaskList taskList;
+    private final UserInterface ui;
+
+    /**
+     * Constructs a Ducky chatbot instance with the specified file path for persistent data.
```

Comment:
Good javadoc, remember to add an empty line between description and parameter section